teuthology/orchestra/run.py

```diff
@@ -34,30 +34,33 @@ class RemoteProcess(object):
         # for orchestra.remote.Remote to place a backreference
         'remote',
         'label',
+        'scan_tests_errors',
         ]
 
     deadlock_warning = "Using PIPE for %s without wait=False would deadlock"
 
     def __init__(self, client, args, check_status=True, hostname=None,
-                 label=None, timeout=None, wait=True, logger=None, cwd=None):
+                 label=None, timeout=None, wait=True, logger=None, cwd=None, scan_tests_errors=[]):
```

Comment:
the default for scan_tests_for_errors should be None rather than an empty list -- due to the way python initializes parameters, using [] or {} as defaults ends up making the variable persistent across invocations of the function, which is usually not what you want.

More detail: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Interesting, thanks for explaining that! Fixed it.